apps/calendar/js/bridge.js

```diff
@@ -91,10 +94,70 @@ var persistEvent = co.wrap(function *(event, action, capability) {
   return provider[action + 'Event'](event.data);
 });
 
-exports.getSetting = co.wrap(function *(id) {
-  var settingsStore = core.storeFactory.get('Setting');
-  return yield settingsStore.getValue(id);
-});
+exports.getSetting = function(id) {
```

Comment:
building the `<select>` for modify event (we only use the setting to mark the default value as `checked`) after user changes the value or the UI is displaying one value it shouldn't update automatically.. (on the open PRs this is the only so far)